src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java

```diff
@@ -233,17 +364,29 @@ public boolean addAllocateChannelsReq(
             ColibriConferenceIQ.Content requestContent
                 = request.getOrCreateContent(contentName);
 
-            ColibriConferenceIQ.ChannelCommon requestChannel
-                = mediaType != MediaType.DATA
-                        ? new ColibriConferenceIQ.Channel()
-                        : new ColibriConferenceIQ.SctpConnection();
+            ColibriConferenceIQ.ChannelCommon requestChannel;
+            if (mediaType == MediaType.DATA)
+            {
+                requestChannel = new ColibriConferenceIQ.SctpConnection();
+            }
+            else if (octoRelayIds != null)
+            {
+                requestChannel = new ColibriConferenceIQ.OctoChannel();
+            }
+            else
+            {
+                requestChannel = new ColibriConferenceIQ.Channel();
+            }
 
-            requestChannel.setEndpoint(endpointName);
+            if (endpointId != null)
+            {
+                requestChannel.setEndpoint(endpointId);
+            }
             requestChannel.setInitiator(peerIsInitiator);
 
-            if (useBundle)
+            if (useBundle && endpointId != null)
             {
-                requestChannel.setChannelBundleId(endpointName);
+                requestChannel.setChannelBundleId(endpointId);
```

Comment:
no bundle id for octo channels because we're not using ice there, right?

Reply:
Correct